pyproject.toml

```diff
@@ -34,13 +34,14 @@ classifiers = [
     "Programming Language :: Python :: 3 :: Only",
     "Programming Language :: Python :: Implementation :: CPython"
 ]
+
 dependencies = [
     "ECmean4==0.1.15", 
 ]
 
 [project.optional-dependencies]
 core = [
-    "aqua-core @ git+https://github.com/DestinE-Climate-DT/AQUA.git"
+    "aqua-core>=0.20.0",
```

Comment:
sorry, this is wrong I feel. we do not have AQUA on pypi so it will not work when you do not apply a non editable installation

Reply:
We can use this when we will have aqua-core on pypi